backend/src/test/kotlin/com/woowahan/campus/zzimkkong/domain/DayOfWeeksTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.woowahan.campus.zzimkkong.domain
+
+import io.kotest.assertions.assertSoftly
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.result.beSuccess
+
+class DayOfWeeksTest : StringSpec({
+
+    "요일명이 올바르면 정상적으로 생성" {
+        assertSoftly {
+            beSuccess(DayOfWeeks.create("MONDAY"))
+            beSuccess(DayOfWeeks.create("TUESDAY"))
+            beSuccess(DayOfWeeks.create("WEDNESDAY"))
+            beSuccess(DayOfWeeks.create("THURSDAY"))
+            beSuccess(DayOfWeeks.create("FRIDAY"))
+            beSuccess(DayOfWeeks.create("SATURDAY"))
+            beSuccess(DayOfWeeks.create("SUNDAY"))
+        }
+    }
```

Comment:
ParameterizedTest 같은거 찾아봤는데 의존성 받아야해요


---

https://veluxer62.github.io/tutorials/getting-started-with-kotest/
Data Driven Testing 부분

Reply:
List forEach를 활용하셔도 됩니다
```kotlin
DayOfWeeks.entries.forEach {
    // assertion
}
```

https://kotest.io/docs/framework/writing-tests.html#dynamic-tests

Reply:
감사합니다 레아! 😎

```suggestion
    "요일명이 올바르면 정상적으로 생성" {
       DayOfWeeks.entries.forEach {
            assertSoftly { beSuccess(DayOfWeeks.create(it.name)) }
        }
    }
```
이렇게 바꾸면 되겠네요

Reply:
ㄱㅅ해요